Godeps/_workspace/src/k8s.io/kubernetes/pkg/apis/extensions/install/install.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/golang/glog"
 
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
I'd expect this to get some serious push back.  Which parts are they dependent upon?